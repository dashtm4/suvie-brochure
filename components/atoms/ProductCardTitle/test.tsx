import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ProductCardTitle } from '.'

describe('<ProductCardTitle />', () => {
  const children = 'Test children'

  it('should renders correct <ProductCardTitle />', () => {
    const wrapper = shallow(<ProductCardTitle>{children}</ProductCardTitle>)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
