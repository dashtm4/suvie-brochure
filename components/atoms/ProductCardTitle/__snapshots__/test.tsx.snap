// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductCardTitle /> should renders correct <ProductCardTitle /> 1`] = `
<h3
  className="font-serif font-normal tracking-tight font-secondary text-center mb-2"
  style={
    Object {
      "fontSize": "2rem",
    }
  }
>
  Test children
</h3>
`;
